src/main/java/org/motechproject/prevac/service/impl/VisitRescheduleServiceImpl.java

```diff
@@ -96,8 +101,6 @@ public VisitRescheduleDto saveVisitReschedule(VisitRescheduleDto visitReschedule
             checkNumberOfPatients(visitRescheduleDto, clinic);
```

Comment:
If we are setting actual date, it means visit already took place, so I don't think we should check number of patients